src/Modules/Profile/Commands.Profile/Commands.Profile.csproj

```diff
@@ -33,6 +33,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
```

Comment:
You many want to make a note that this isn't a "real" dependency and simply a way to be the first to claim the dependency in the app domain (assuming another cmdlet hasn't beat you to it). IMO, it would be better to put this in a project entirely devoted to being loaded first and have all the projects reference it.

Reply:
I'll consider this for a future iteration. I want to note that the Profile module does require this dependency for one of its NuGet packages and the Profile module already functions as a project that will be loaded first since all other cmdlets depend upon it.